ale/drivers/base.py

```diff
@@ -120,12 +177,13 @@ def _compute_ephemerides(self):
 
 
     @property
-    @abstractmethod
-    def metakernel(self):
-        pass
+    def line_exposure_duration(self):
+        try:
+            return self._label['LINE_EXPOSURE_DURATION'].value * 0.001  # Scale to seconds
+        except:
+            return np.nan
```

Comment:
Why is this in a try except?

Reply:
Some labels do not have LINE_EXPOSURE_DURATION. We definitely need a better way to handle this case.

Reply:
Different instruments also use different units on this :/